files/image_config/interfaces/interfaces.j2

```diff
@@ -5,6 +5,11 @@
 # file: /etc/network/interfaces
 #
 {% endblock banner %}
+{% if (MGMT_VRF_CONFIG) and (MGMT_VRF_CONFIG['vrf_global']['mgmtVrfEnabled'] == "true") %}
+auto mgmt
+iface mgmt
+    vrf-table 4999
```

Comment:
in the previous discussion, we ask this to be changed to 1000, why change this to 4999?

Reply:
Guohan, while unit testing with either 1000 or any vrf table id less than 1000, we were getting the following error **"Error is : Nov 04 04:18:31 sonic networking[6426]: error: eth0: mgmt: vrf table id 999 out  of reserved range [1001,5000]**. Looking at the ifupdown2 code  the reserved range of vrf table id's are 1001 - 5000. So I changed it to 4999. Below is the snippet from the ifupdown2 code where they are initializing. 
    VRF_TABLE_START = 1001
    VRF_TABLE_END = 5000

Reply:
why not 5000 then?

Reply:
changed to 5000